net/fail2ban/Makefile

```diff
@@ -14,12 +14,14 @@ PKG_SOURCE_URL:=https://codeload.github.com/fail2ban/fail2ban/tar.gz/$(PKG_VERSI
 PKG_HASH:=383108e5f8644cefb288537950923b7520f642e7e114efb843f6e7ea9268b1e0
 
 PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
-PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE:=GPL-2.0-only
```

Comment:
Where did you see it is GPL2 only? According to the [readme](https://github.com/fail2ban/fail2ban/blob/095aeda8407b433098df35424cde2764a09566a6/README.md?plain=1#L120-L121) it is GPL2 or later.

Reply:
fixed... thanks